samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -326,8 +326,10 @@ - (UIImage*)transformImage:(UIImage*)image {
     // create CGImageRef from vImage_Buffer output
     CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
 
-    CGContextRef ctx = CGBitmapContextCreate(
-        outBuffer.data, outBuffer.width, outBuffer.height, 8, outBuffer.rowBytes, colorSpace, (CGBitmapInfo)kCGImageAlphaNoneSkipLast);
+    const CGBitmapInfo bitmapInfo = kCGImageAlphaNoneSkipLast | kCGBitmapByteOrder32Little;
```

Comment:
Thats perfect.  There are other files where this change as well.  

---
In reply to: [71264537](https://github.com/Microsoft/WinObjC/pull/641#discussion_r71264537) [](ancestors = 71264537)